subsys/net/lib/sockets/getaddrinfo.c

```diff
@@ -19,6 +20,13 @@ LOG_MODULE_REGISTER(net_sock_addr, CONFIG_NET_SOCKETS_LOG_LEVEL);
 
 #if defined(CONFIG_DNS_RESOLVER)
 
+/* Helper macros which take into account the fact that ai_family as passed
+ * into getaddrinfo() may take values AF_INET, AF_INET6, or AF_UNSPEC, where
+ * AF_UNSPEC means resolve both AF_INET and AF_INET6.
+ */
+#define RESOLVE_IPV4(ai_family) (IS_ENABLED(CONFIG_NET_IPV4) && (ai_family) != AF_INET6)
```

Comment:
come on: 80 chars per-line. Your editor should help you on that.

Reply:
fixed